e2e/autocomplete.e2e-spec.ts

```diff
@@ -15,17 +16,17 @@ describe('Autocomplete', () => {
     SkyHostBrowser.setWindowBreakpoint('lg');
   });
 
-  function activateDropdown(done: DoneFn): void {
+  function activateDropdown(): void {
     const input = element(by.css('#favorite-color-reactive'));
     input.value = 'r';
     input.click();
     browser.actions().sendKeys('r').perform();
 
-    browser.wait(() => {
-      return browser.isElementPresent(
-        element(by.css('.sky-dropdown-item'))
-      );
-    });
+    browser.wait(
```

Comment:
Are all these waits here for the animation or loading results? Seems like we could mock those responses or kill the animations and not have to do any waiting. Is that possible?

Reply:
They are waiting for the results to appear (there is a debounce time applied to the key events). This was rewritten from what was there before to accommodate the new overlays.